contracts/smart-contract-wallet/MaliciousAccount.sol

```diff
@@ -208,16 +208,13 @@ contract MaliciousAccount is
                     // Payment info
                     refundInfo,
                     // Signature info
-                    nonces[batchId]
+                    nonces[0]++
```

Comment:
should this be 1 as well if changes made elsewhere

Reply:
fixed